modules/ols-filtering-and-redacting-information.adoc

```diff
@@ -5,13 +5,20 @@
 [id="ols-filtering-and-redacting-information_{context}"]
 = Filtering and redacting information
 
-You can configure {ols-long} to filter or redact information from being sent to the LLM provider. The following example shows how to modify the `OLSConfig` file to redact IP addresses.   
+Configure {ols-long} to filter or redact sensitive information, preventing it from being sent to the LLM provider. 
 
 [NOTE]
 ====
 You should test your regular expressions against sample data to confirm that they are catching the information you want to filter or redact, and that they are not accidentally catching information you do not want to filter or redact. There are several third-party websites that you can use to test your regular expressions. When using third-party sites, you should practice caution with regards to sharing your private data. Alternatively, you can test the regular expressions locally using Python. In Python, it is possible to design very computationally-expensive regular expressions. Using several complex expressions as query filters can adversely impact the performance of {ols-long}.
 ====
 
+This example shows how to modify the `OLSConfig` custom resource (CR) file to redact IP addresses, but you can also filter or redact other kinds of sensitive information.   
+
+[NOTE]
+====
+If you configure filtering or redacting in the `OLSConfig` CR file, and you configure `introspectionEnabled` in a Model Context Protocol (MCP) server, content that tools return is not filtered and is visible to the LLM.
```

Comment:
`introspectionEnabled` isn't configured "in an MCP server" -- that's what enables the OCP MCP server.